logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCase.kt

```diff
@@ -60,6 +61,9 @@ class GetNotificationsUseCaseImpl(
     override suspend operator fun invoke(): Flow<List<LocalNotificationConversation>> {
         return observeRegularNotifications()
             .combine(observeConnectionRequests()) { messages, connections -> messages.plus(connections) }
+            .combine(ephemeralNotificationsManager.observeEphemeralNotifications()) { messages, ephemeralNotifications ->
```

Comment:
I'm not an expert on this kotlin operator. Would this `combine` method only zip together the normal `messages` notifications and `ephemeralNotifications` or is this `messages` value the result of combining the normal `messages` and `connections` notifications from above?

Reply:
It is the second case, each operation is chained, so the ouput of the first combine (joining messages and connections) will be used as an input on the second one :D